front/app/containers/Admin/communityMonitor/components/LiveMonitor/components/HealthScoreWidget/utils.ts

```diff
@@ -0,0 +1,84 @@
+import { colors, Locale } from '@citizenlab/cl2-component-library';
+
+import { CommunityMonitorSentimentScoreAttributes } from 'api/community_monitor_scores/types';
+
+import { QuarterlyScores } from './types';
+
+export const categoryColors = {
+  overall: colors.green400,
+  quality_of_life: colors.teal400,
+  service_delivery: colors.blue400,
+  governance_and_trust: colors.brown,
+  other: colors.grey700,
+};
+
+// transformSentimentScoreData:
+// Transform the sentiment score data object into a more usable format with arrays.
+export function transformSentimentScoreData(
```

Comment:
Might be worth a test for this method

Reply:
I've already implemented some unit tests for this one! :relaxed::+1: